docs/pages/enroll-resources/server-access/guides/encrypted-session-recordings.mdx

```diff
@@ -0,0 +1,269 @@
+resource dynamically using `tctl`.
+
+You can edit the Auth Service configuration file as follows:
+
+```yaml
+# snippet from teleport.yaml
+auth_service:
+  session_recording_config:
+    encryption:
+      enabled: yes
+```
+
+You can also edit the `session_recording_config` resource using
+`tctl edit session_recording_config`:
+
+```yaml
+# snippet from session_recording_config
+spec:
+  encryption:
+    enabled: yes
+```
+
+If you are already configuring session recording through the configuration file
+it is recommended that you enable encryption there as well, as modifications
+made through `tctl` will be overwritten when the Teleport Auth Service
+restarts.
+
+### Optional step: Configure key storage backend
+
+Session recording encryption keys are provisioned using the same key storage
+backend configured for CAs. Software keys stored in Teleport's backend storage
+are the default, but other backends can also be configured:
+
+- [AWS KMS](../../../zero-trust-access/deploy-a-cluster/aws-kms.mdx)
+- [Google Cloud KMS](../../../zero-trust-access/deploy-a-cluster/gcp-kms.mdx)
+- [HSM](../../../zero-trust-access/deploy-a-cluster/hsm.mdx)
+
+It is important that all Teleport Auth Services have access to the same keys
+and key storage backend to ensure all recorded sessions are always replayable.
+
+## Step 2/2. Confirm new recordings are encrypted
+
+It is possible to confirm that session recordings are now encrypted by
+capturing a new recording and downloading the resulting `.tar` file from your
+[audit sessions backend](../../../reference/architecture/session-recording.mdx#storage).
+Replaying the session file directly with `tsh play` should fail and replaying
+using the session ID should succeed.
+
+## Manual Encryption Key Management
+
+<Admonition type="note">
+Manual encryption key management places all responsibility and complexity of
+managing keys on the administrator. This comes with much more risk of a
+misconfiguration impacting Teleport's ability to record or replay sessions.
+It's for this reason that Teleport Cloud does not allow use of manual
+encryption key management. Consider using the default automatic key management
+unless your usage explicitly requires manual key management.
+</Admonition>
+
+By default, Teleport will automatically provision and manage session recording
+encryption keys. However there may be cases where automatic management is not
+possible or desired, such as environments where Teleport lacks sufficient
+privileges to manage keys directly. For this reason, Teleport can be configured
+to use externally managed keys.
+
+### Configuration
+
+```yaml
+# snippet from session_recording_config
+encryption:
+  enabled: yes
+  manual_key_management:
+    enabled: yes
+    active_keys:
+      - type: pkcs11
+        label: 'session_recordings_002'
+    rotated_keys:
+      - type: pkcs11
+        label: 'session_recordings_001'
+```
+
+This example enables `manual_key_management` with one active key and one
```

Comment:
I would put this paragraph before the snippet so the user knows why we're including it.